data/darktableconfig.xml.in

```diff
@@ -2318,4 +2318,11 @@
     <shortdescription>overwrite the screen's ppd setting</shortdescription>
     <longdescription>if this value is > 0.0 then it is used as the screen's ppd setting which is used for HiDPI support</longdescription>
   </dtconfig>
+  <dtconfig>
+    <name>plugins/lighttable/preview/full_size_preload_count</name>
+    <type>int</type>
+    <default>3</default>
```

Comment:
Yep, you're right on the cost which is why I asked if you had tried 1 or 2. The number should be as low as possible while still getting fluid previews. 3 sounds fine too though if that's what's needed.